packages/ecommerce-app-base/src/AppConfig/OrchestrationEapNote/OrchestrationEapNote.tsx

```diff
@@ -13,19 +13,18 @@ const styles = {
 
 interface Props {
   name: string;
-  hideOrchestrationEapNote: boolean;
+  setHideOrchestrationEapNote: (hideOrchestrationEapNote: boolean) => void;
 }
 
-export const OrchestrationEapNote = ({ hideOrchestrationEapNote, name }: Props) => {
-  const [, setHideOrchestrationEapNote] = useState(false);
+export const OrchestrationEapNote = ({ setHideOrchestrationEapNote, name }: Props) => {
   const title = `The ${name} app supports External references`;
+
+  const handleClose = () => {
+    setHideOrchestrationEapNote(true);
+  };
+
```

Comment:
Why can't this component handle the state locally?

Reply:
Because otherwise it does never get closed, the app state needs to be aware of it. (It was previously like that we realized that there is a bug. 🐞 )